docs/app/pages/components/components-sections/facets/dynamic-facets-ng1/dynamic-facets-ng1.components.html

```diff
@@ -60,6 +69,11 @@
 <p>Set the <code>placeholder</code> attribute to a string property to set the placeholder text that will be shown in the text
     search input control.</p>
 
+<h4>Customizing the Dynamic Facet Option Template</h4>
```

Comment:
Can you remove this heading? It gives the appearance that the sample code only applies to the template subsection. The current documentation standard is to list attributes in a tabular format for clarity, but since this is a deprecated component I don't think it's worth making that change now.

Reply:
Done